kitchen.yml

```diff
@@ -0,0 +1,75 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  product_name: cinc
```

Comment:
should we be testing against cinc or chef-infra

Reply:
We should be using chef for Sous Chefs.